packages/core/src/polyfills/custom-event-polyfill.js

```diff
@@ -0,0 +1,12 @@
+;(function() {
```

Comment:
we will need to check that window exists too - webcomponents only loads in browser but core loads everywhere (browser, server, native, possibly even webworker)

so just a `if (typeof window === 'undefined') { return }` should suffice

Reply:
it's in the last commit, but will update to use this syntax

Reply:
oh oops I missed that (clicked via email), what you have totally works